python/langsmith/_internal/_background_thread.py

```diff
@@ -141,7 +138,10 @@ def _tracing_thread_drain_compressed_buffer(
                     f"size_limit_bytes must be nonnegative; got {size_limit_bytes}"
                 )
 
-            if (size_limit_bytes is None or current_size < size_limit_bytes) and (
+            if (
+                size_limit_bytes is None
+                or pre_compressed_size < size_limit_bytes
```

Comment:
So before we were batching based on the compressed size, now we're going by the uncompressed size?

This will be smaller batches in general right?